rbx_binary/src/deserializer.rs

```diff
@@ -861,7 +861,36 @@ impl<R: Read> BinaryDeserializer<R> {
                     });
                 }
             },
-            Type::Ref => {}
+            Type::Ref => match canonical_type {
+                VariantType::Ref => {
+                    let mut differences = vec![0; type_info.referents.len()];
+                    chunk.read_interleaved_i32_array(&mut differences)?;
```

Comment:
We have a function for reading referents here that we should probably use instead
https://github.com/rojo-rbx/rbx-dom/blob/master/rbx_binary/src/core.rs#L127